cmd/main.go

```diff
@@ -208,10 +210,12 @@ func setupProxmoxClient(ctx context.Context, logger logr.Logger) (capmox.Client,
 	}
 
 	httpClient := &http.Client{Transport: tr}
-	return goproxmox.NewAPIClient(ctx, logger, ProxmoxURL,
-		proxmox.WithHTTPClient(httpClient),
-		proxmox.WithAPIToken(ProxmoxTokenID, ProxmoxSecret),
-	)
+	opts := []any{proxmox.WithHTTPClient(httpClient), proxmox.WithAPIToken(ProxmoxTokenID, ProxmoxSecret)}
+	if strings.ToLower(ProxmoxReserveOnlyRunning) == "true" {
+		opts = append(opts, goproxmox.WithCalcOnlyRunning())
```

Comment:
Wouldn't be better to set it as field in the cluster/machine spec?

Reply:
I would tend to agree.

Reply:
> Wouldn't be better to set it as field in the cluster/machine spec?
Maybe on Cluster scope, on mashine scope need special config structure for each physical node in physical proxmox Cluster.  
proxmoxmachines.infrastructure.cluster.x-k8s.io is virtualMashine and very strange set this option here (not For Physical mashine|node)